pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -39,18 +39,18 @@ const (
 	maxAllowableNUMANodes = 8
 )
 
-//Manager interface provides methods for Kubelet to manage pod topology hints
+// Manager interface provides methods for Kubelet to manage pod topology hints
 type Manager interface {
-	//Manager implements pod admit handler interface
+	// Manager implements pod admit handler interface
```

Comment:
hm... golint would want the comment to start with the field name. Can you fix it since you are looking into it